### PR TITLE
css updates search

### PR DESCRIPTION
**Pull request by davelahaye**
Minor changes to layout including moving the title bar into index.
Switched to css grid to display search results.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -42,14 +47,15 @@ export default function Index() {
           <SearchResult key={`SearchResult-${index}`}>
             <ResultHeader>
               {
-                index === 0
+                index === 1
```
Comment (brentmclark) [inline pages/index.js:50]: You want to select the second one, not the first one?

File: pages/index.js [hunk 2]
```diff
@@ -58,14 +64,32 @@ export default function Index() {
...
+const PageTitle = styled.h1`
+  font-size: 24px;
+  font-weight: 300;
+  margin-block-start: 0;
+  letter-spacing: 0;
+`
+
 const SearchContainer = styled.form`
-  width: 100%;
+  width: 100% - 60px;
```
Comment (brentmclark) [inline pages/index.js:84]: Does this work without a `calc()`?  I haven't seen this syntax before.

File: pages/index.js [hunk 3]
```diff
@@ -28,7 +29,11 @@ export default function Index() {
 
   return (
     <PrimaryLayout>
+      <TitleBar>
+        <PageTitle>Card Search</PageTitle>
+      </TitleBar>
```
Comment (brentmclark) [inline pages/index.js:34]: I've been giving this particular change some thought, and want to get your take on moving it back to the template, but grabbing it's value from a `pageTitle` prop it something like that. 

It seems unnecessary to make each page implement the components for its own page title.

**Comment by brentmclark** [comment]
Looks great overall, a few questions but nothing show-stopping.  Merge away!

File: pages/index.js (hunk 1 above)
Comment (davelahaye) [reply pages/index.js:50]: Was playing around to learn how it was working... I'll change it back.

File: pages/index.js (hunk 2 above)
Comment (davelahaye) [reply pages/index.js:84]: Haha!  Good catch.  Nope, just removed the variables used to calculate and "missed a spot".... Looks like it still respected the first value.  Which is nice!

File: pages/index.js (hunk 3 above)
Comment (davelahaye) [reply pages/index.js:34]: I agree.  This layout is a little different than the rest I am working through and I didn't want to jank things up so I pulled it out for now.

Comment (davelahaye) [reply pages/index.js:34]: We should revisit when we see a repeatable pattern.  Thoughts?

Comment (brentmclark) [reply pages/index.js:34]: That seems perfectly reasonable, yep.

**Pull request merged**